src/dbnode/server/server.go

```diff
@@ -391,6 +391,10 @@ func Run(runOpts RunOptions) {
 			scope.SubScope("tag-decoder-pool")))
 	tagDecoderPool.Init()
 
+	// Pass nil for block.LeaseVerifier for now and it will be set after the
+	// db is constructed.
```

Comment:
Maybe also add that this needs to be initialized here so that it can be propagated down before the db is created.

Reply:
sure